samples/bluetooth/encrypted_advertising/peripheral/src/common.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/ead.h>
+
+#ifndef __EAD_SAMPLE_COMMON_H
+#define __EAD_SAMPLE_COMMON_H
+
+struct key_material {
+	uint8_t session_key[BT_EAD_KEY_SIZE];
+	uint8_t iv[BT_EAD_IV_SIZE];
+} __packed;
+
+#define CUSTOM_SERVICE_TYPE BT_UUID_128_ENCODE(0x2e2b8dc3, 0x06e0, 0x4f93, 0x9bb2, 0x734091c356f0)
```

Comment:
why is it under peripheral/ if it's called `common.h`? it seems like it's also duplicated for the central.